src/SMTP.php

```diff
@@ -372,7 +372,7 @@ public function connect($host, $port = null, $timeout = 30, $options = [])
         // SMTP server can take longer to respond, give longer timeout for first read
         // Windows does not have support for this timeout function
         if (strpos(PHP_OS, 'WIN') !== 0) {
-            $max = (int)(ini_get('max_execution_time'));
+            $max = (int) (ini_get('max_execution_time'));
```

Comment:
You don't need those parentheses around the ini_get anyway.

Reply:
ok